tests/metadata/metaschema/draft07/test_schema.py

```diff
@@ -1,15 +0,0 @@
-def test_if_schema_json_loads_successfully():
```

Comment:
Why did you remove these files in a commit with the title "Rename metadata to oemetadata" and not in a subsequent commit?

Reply:
ok I understood, renaming the file ...